GI01SUMO/GI01SUMO_R00011_ingest.csv

```diff
@@ -0,0 +1,86 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl16/syslog/*syslog.log,GI01SUMO-RID16-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl16/optaa2/*.optaa2.log,GI01SUMO-RID16-01-OPTAAD000,recovered_host,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl16/flort2/*.flort2.log,GI01SUMO-RID16-02-FLORTD000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl16/ctdbp/*ctdbp.log,GI01SUMO-RID16-03-CTDBPF000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/instruments/dcl16/CTDBP_sn*/*.hex,GI01SUMO-RID16-03-CTDBPF000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl16/velpt/*.velpt.log,GI01SUMO-RID16-04-VELPTA000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/instruments/dcl16/VELPT_sn*/*.aqd,GI01SUMO-RID16-04-VELPTA000,recovered_inst,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl16/pco2w/*.pco2w.log,GI01SUMO-RID16-05-PCO2WB000,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/instruments/dcl16/PCO2W_sn*/*V.txt,GI01SUMO-RID16-05-PCO2WB000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl16/dosta2/*dosta2.log,GI01SUMO-RID16-06-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.suna.suna_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl16/nutnr2/*.nutnr2.log,GI01SUMO-RID16-07-NUTNRB000,recovered_host,Available,
+mi.dataset.driver.suna.suna_instrument_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/instruments/dcl16/NUTNR_sn*/DAT/*.CSV,GI01SUMO-RID16-07-NUTNRB000,recovered_inst,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl16/spkir2/*.spkir2.log,GI01SUMO-RID16-08-SPKIRB000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/cg_data/dcl11/imm/adcp/adcp_*.DAT,GI01SUMO-RII11-02-ADCPSN010,recovered_host,Available,
+#mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/GI01SUMO/R00011/instruments/dcl11/imm/ADCP_sn*/_RDI*.000,GI01SUMO-RII11-02-ADCPSN010,recovered_inst,Not Expected,Data could not be downloaded from instrument
```

Comment:
The file on RDA [noRecoveredData.txt](https://rawdata.oceanobservatories.org/files/GI01SUMO/R00011/instruments/dcl11/imm/ADCP_sn21717/noRecoveredData.txt) is empty